src/NuGet.Core/NuGet.CommandLine.XPlat/ListPackage/ListPackageConsoleRenderer.cs

```diff
@@ -0,0 +1,235 @@
+                            case ReportType.Default:
+                                Console.WriteLine(string.Format(CultureInfo.CurrentCulture, "   [{0}]: " + Strings.ListPkg_NoPackagesForFramework, frameworkPackages.Framework));
+                                break;
+                        }
+
+                        Console.ResetColor();
+                    }
+                    else
+                    {
+                        // Print name of the framework
+                        Console.ForegroundColor = ConsoleColor.Blue;
+                        Console.WriteLine(string.Format(CultureInfo.CurrentCulture, "   [{0}]: ", frameworkPackages.Framework));
+                        Console.ResetColor();
+
+                        // Print top-level packages
+                        if (frameworkTopLevelPackages?.Any() == true)
+                        {
+                            var tableHasAutoReference = false;
+                            var tableToPrint = ProjectPackagesPrintUtility.BuildPackagesTable(
+                                frameworkTopLevelPackages, printingTransitive: false, listPackageArgs, ref tableHasAutoReference);
+                            if (tableToPrint != null)
+                            {
+                                ProjectPackagesPrintUtility.PrintPackagesTable(tableToPrint);
+                            }
+                        }
+
+                        // Print transitive packages
+                        if (listPackageArgs.IncludeTransitive && frameworkTransitivePackages?.Any() == true)
+                        {
+                            var tableHasAutoReference = false;
+                            var tableToPrint = ProjectPackagesPrintUtility.BuildPackagesTable(
+                                frameworkTransitivePackages, printingTransitive: true, listPackageArgs, ref tableHasAutoReference);
+                            if (tableToPrint != null)
+                            {
+                                ProjectPackagesPrintUtility.PrintPackagesTable(tableToPrint);
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        private static void PrintSources(IEnumerable<PackageSource> packageSources)
+        {
+            foreach (var source in packageSources)
+            {
+                Console.WriteLine("   " + source.Source);
+            }
+        }
+
+        private static void PrintProblems(IEnumerable<ReportProblem> problems, ListPackageArgs listPackageArgs)
+        {
+            if (problems == null)
+            {
+                return;
+            }
+
+            foreach (ReportProblem problem in problems)
+            {
+                switch (problem.ProblemType)
+                {
+                    case ProblemType.Warning:
+                        if (problem.Text.StartsWith("You are running the 'list package' operation with an 'HTTP' source"))
```

Comment:
This string gets localized so this won't work. 

I think my feedback was that we can `logwarning` in both cases and not special case.

Reply:
> This string gets localized so this won't work.
> 
> I think my feedback was that we can `logwarning` in both cases and not special case.

Sure, I can use `logwarning` both cases, but we'll get extra `warn : ` which not there before my change. Is it ok? Not really breaking change, but still user experince change. Do you think it's acceptable?
![image](https://user-images.githubusercontent.com/8766776/199813775-b2c1b912-1002-49cf-971b-7fc927a7726c.png)

Reply:
I think it's ok. That should have been a warning anyways.

Reply:
> I think it's ok. That should have been a warning anyways.

Made both of them single `logwarning`.